### ISSUE TITLE
Question: multi-line command substitution in script vs interactive

### ISSUE THREAD

**Issue by patrickf3139** [fish-shell/fish-shell#6316]
In one of my own fish scripts, I have a line that looks like
```fish
set variable (
        command1 |
        command2 |
        command3
)
```
So I was really surprised that when I tried to type it out in the interactive shell that this was printed
```
 ~  set variable (                                                                                                   
fish: Unexpected end of string, expecting ')'
set variable (
```
Is it expected that in the interactive shell you can't do multi-line command substitution?

**Comment by faho** [comment]
No, it's not expected, the editor is overzealous here.

**Comment by patrickf3139** [comment]
Woohoo found another bug! Thanks for all your work on the fish shell faho!

**Comment by ridiculousfish** [comment]
Seems reasonable. Fixed as e18fd3cddb57e9421b536405ff6e17abde469d8e. Thanks for filing!
